config/locales/en.yml

```diff
@@ -1801,6 +1801,11 @@ en:
 
   notifications:
     title: "Notifications"
```

Comment:
I prefer to use the detailed settings path for these locales, e.g. `users.settings.account.preferences.edit.tips`. Tips are **not** part of notifications, but a separate section of settings.